src/pages/AdminSettings/AdminMembers/ActivationInfo/AutoActivateMembers/AutoActivateMembers.tsx

```diff
@@ -24,7 +35,7 @@ function AutoActivateMembers() {
         <Toggle
           dataMarketing="auto-acitvate-members"
           label={data?.planAutoActivate ? 'On' : 'Off'}
-          value={data?.planAutoActivate}
+          value={!!data?.planAutoActivate}
```

Comment:
Do we need the boolean cast for a certain reason?

Reply:
Yep, we need this here because `data?.planAutoActivate` is of type `boolean | null` and `value` only accepts booleans

Reply:
got it makes sense!